### PR TITLE
Update the Application mobility and remove the interactive tutorials

### PR DESCRIPTION
**Pull request by anandrajak1**
# Description
This PR is to address documentation update for Application mobility  and remove the Intractive tutorials

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1435 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?

### CODE REVIEW

File: content/docs/applicationmobility/_index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Description: >
 ---
 
 {{% pageinfo color="primary" %}}
-Application Mobility is available with [APEX Navigator for Kubernetes](https://www.dell.com/en-ca/dt/apex/storage/public-cloud/navigator.htm#kubernetes)
+Coming Soon: Application Mobility will be available with CSM early next year (2025)
```
Comment (AronAtDell) [inline content/docs/applicationmobility/_index.md:10]: Change this to be more professional - "We are pleased to announce that Application Mobility will be available with Container Storage Modules starting early next year (2025)."

**Review by AronAtDell — APPROVED** [review]
Apart from my comment looks good. Thanks!

File: content/docs/applicationmobility/_index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Description: >
 ---
 
 {{% pageinfo color="primary" %}}
-Application Mobility is available with [APEX Navigator for Kubernetes](https://www.dell.com/en-ca/dt/apex/storage/public-cloud/navigator.htm#kubernetes)
+We are pleased to announce that Application Mobility will be available with Container Storage Modules starting early next year (2025).
```
Comment (sharmilarama) [inline content/docs/applicationmobility/_index.md:10]: This message needs to be updated in the live docs as well.

Comment (sharmilarama) [reply content/docs/applicationmobility/_index.md:10]: Same message should be updated in the Release notes page as well (https://dell.github.io/csm-docs/docs/applicationmobility/release/)

Comment (anandrajak1) [reply content/docs/applicationmobility/_index.md:10]: Yes, I will have separate PR for 1.11.1 branch once its merged to main.

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**